src/main/scala/me/jeffshaw/digitalocean/DelayedFuture.scala

```diff
@@ -0,0 +1,23 @@
+package me.jeffshaw.digitalocean
+
+import java.util.{TimerTask, Timer}
+
+import scala.concurrent.{Promise, Future}
+import scala.concurrent.duration.Duration
+
+object DelayedFuture {
+
+  def after[T](duration: Duration)(future: => Future[T]) = {
+    val p = Promise[T]()
+
+    val timer = new Timer(false)
```

Comment:
Shouldn't this be a daemon thread? If the JVM is shutting down, we probably don't want to wait for an http request to finish.